webpack/assets/javascripts/react_app/components/hosts/storage/vmware/controller/disk/index.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { Button } from 'react-bootstrap';
+import Select from '../../../../../common/forms/Select';
+import Checkbox from '../../../../../common/forms/Checkbox';
+import TextInput from '../../../../../common/forms/TextInput';
+import './disk.scss';
+
+const Disk = (
+  {
+    removeDisk,
+    updateDisk,
+    name,
+    storagePod,
+    datastores,
+    storagePods,
+    dataStore,
+    size,
+    thinProvision,
+    eagerZero
+  }
+) => {
+  const renderOptions = (arr) => arr.map(attribute => {
+    const key = Object.keys(attribute)[0];
+    const value = Object.values(attribute)[0];
+
+    return <option key={key} value={key}>{value}</option>;
+  });
+  const selectableStores = renderOptions(datastores);
+  const selectablePods = renderOptions(storagePods);
+
+  return (
+    <div className="disk-container">
```

Comment:
added, do you want to send the possible values the same was as controller_types?